api/src/main/java/org/openmrs/Concept.java

```diff
@@ -652,9 +652,7 @@ public ConceptName getPreferredName(Locale forLocale) {
 		}
 		// fail early if this concept has no names defined
 		if (getNames(forLocale).isEmpty()) {
-			if (log.isDebugEnabled()) {
-				log.debug("there are no names defined for concept with id: " + conceptId + " in the  locale: " + forLocale);
-			}
+			log.debug("The locale is {}.", forLocale);
 			return null;
```

Comment:
Logger message should be the same . It should display the both concept id and locale . With your change it will log only the locale information